react-app/src/features/welcome/default.tsx

```diff
@@ -0,0 +1,173 @@
+import * as Heroicons from "@heroicons/react/24/outline";
+import { Link } from "react-router";
+
+import * as C from "@/components";
+import { Button } from "@/components";
+import { CardWithTopBorder } from "@/components";
+import { FAQ_TAB } from "@/consts";
+import { useHideBanner } from "@/hooks/useHideBanner";
+
+export const Welcome = () => {
+  const isSectionHidden = useHideBanner();
+
+  return (
+    <>
+      <div className="w-full bg-primary p-2 md:p-4">
+        <div className="max-w-screen-xl flex flex-col sm:flex-row sm:items-center gap-4 mx-auto p-4 lg:px-8">
+          <h1>
+            <img src="/onemac-logo.png" alt="onemac" className="p-4 min-w-[400px]" />
+          </h1>
+          <p className="text-center text-white/100 font-light text-xl font-sans">
+            Welcome to the official submission system for paper-based state plan amendments (SPAs)
+            and section 1915 waivers.
+          </p>
+        </div>
+      </div>
+      {/* End Hero Section */}
+      {/* Two Column Main Layout */}
+      <div className="max-w-screen-xl mx-auto p-4 lg:px-8">
+        <div
+          className="m-auto max-w-[767px]"
+          style={{ display: isSectionHidden ? "none" : "block" }}
```

Comment:
We should stick to Tailwind when we can:

```suggestion
          className={cn("m-auto max-w-[767px]", isSectionHidden ? "hidden" : "block" )}
```

Reply:
Good catch. This is some legacy code from the original welcome/index.ts, so this must have been lingering around for a while. Might be worth a check to see where else in our codebase we're still doing it this way.